### PR TITLE
WIP [Magiclysm] Bugfix Itemgroup enchanted misc "prob" to weighting

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

### CODE REVIEW

**Comment by KorGgenT** [comment]
this pr doesn't look right. i would have expected replacing collection with distribution was the right call, instead of moving it all over to square brackets? the game also won't run as is here. i pinged you on the discord but i guess you weren't around at the time.

File: data/mods/Magiclysm/itemgroups/itemgroups.json [hunk 1]
```diff
@@ -294,17 +294,17 @@
...
-      { "item": "mtorch_everburning", "prob": 100 },
-      { "item": "mflask_hip_whiskey", "prob": 100 },
-      { "item": "mtailors_kit", "prob": 100 },
-      { "item": "mspider_box", "prob": 100 },
-      { "item": "cauldron_demon_chitin", "prob": 20 },
-      { "item": "fridge_holding_1", "prob": 12 },
-      { "item": "bag_holding_1", "prob": 8 },
-      { "item": "bag_holding_2", "prob": 1 },
-      { "item": "cauldron_orichalcum", "prob": 100 }
+      [ "item": "heat_cube", 100 ],
```
Comment (ZhilkinSerg) [inline data/mods/Magiclysm/itemgroups/itemgroups.json:297]: That is incorrect syntax.

Comment (Maleclypse) [reply data/mods/Magiclysm/itemgroups/itemgroups.json:296]: Thank you!

File: data/mods/Magiclysm/itemgroups/itemgroups.json [hunk 2]
```diff
@@ -292,19 +292,18 @@
...
+      [ "mkey_opening", 100 ],
+      [ "mtorch_everburning", 100 ],
+      [ "mflask_hip_whiskey", 100 ],
+      [ "mtailors_kit", 100 ],
+      [ "mspider_box", 100 ],
+      [ "cauldron_demon_chitin", 20 ],
+      [ "fridge_holding_1", 12 ],
+      [ "bag_holding_1", 8 ],
+      [ "bag_holding_2", 1 ],
+      [ "cauldron_orichalcum", 100 ]
```
Comment (KorGgenT) [inline data/mods/Magiclysm/itemgroups/itemgroups.json:306]: i'm willing to bet this is totally unecessary. can you revert this part with the removal of collection and check that it still does it in game for me? i want to reduce unecessary churn, and i prefer the object way of representing things in json.

Comment (Maleclypse) [reply data/mods/Magiclysm/itemgroups/itemgroups.json:306]: So revert the changes to the item list and only remove the ‘“subtype”: “collection”’?

**Comment by Maleclypse** [comment]
Suggestion from @KorGgenT  worked.  However, reverting the commits is turning into a nightmare.  Closing this and making the change in forge of wonders PR.

**Pull request closed without merging**